### PR TITLE
Alicia Bremer onboarding

### PR DESCRIPTION
**Pull request by aliciabremer**
# Strategy Onboarding

## TODO List

Onboarding tasks:

- [ ] Task 1: Implement `fibonacci_term()` in `fibonacci.py`
  - All tests in `test_fibonacci.py` should pass, run `pytest` command to check
- [ ] Task 2: Implement `find_min_index()` and `reverse_str_arr()`
  - Write tests that cover the logic that you wrote in `test_array.py`
- [ ] Task 3: Process and visualize sample data in `matplot.py`
  - Read data in CSV using `pandas` or `csv`, handle the missing values, plot using `matplotlib`
  - Include a screenshot of your graph in the PR

Some extra items:

- [ ] Completed [strategy welcome survey](https://forms.gle/xnYBumyAn2mA2Ao68)
- [ ] Added `@e-wai` as a reviewer (top of right column in the "Open a pull request" page)

### CODE REVIEW

**Comment by alicia639** [comment]
@e-wai 

I'm not sure why but I can't seem to add a reviewer.

File: onboarding/matplot.py [hunk 1]
```diff
@@ -5,3 +5,29 @@
...
+
+import pandas as pd
+import matplotlib.pyplot as plt
+%matplotlib inline
+
+df = pd.read_csv('./test_data.csv')
+
+df["0"] = df["0"].fillna(df["0"].mean())
+df["1"] = df["0.1"].fillna(df["0.1"].mean())
+df["0.2"] = df["0.2"].fillna(df["0.2"].mean())
```
[the remark this answers is not in the document]
Comment (alicia639) [reply onboarding/matplot.py:17]: The "0.2" was the title of the csv column.

File: onboarding/array.py [hunk 2]
```diff
@@ -7,9 +7,18 @@ def find_min_index(arr):
     @param array of integers
     @returns integer index of minimum integer
     '''
+    if len(arr)==0:
+        raise TypeError()
 
-    # insert code here
-    pass
+    min = 0
```
Comment (e-wai) [inline onboarding/array.py:13]: Maybe call this `min_index` for clarity?

**Review by e-wai — APPROVED** [review]
Looks solid to me, nice job!

**Pull request closed without merging**